_posts/2022-08-05-nx-plugins.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Announcing our open-source project, Nx Plugins
+published: true
+description: Nx Plugins is a Nx mono repo which consists of plugins which supports the Nx eco-system.
+author: Farhan Patwary, Prabhat Thapa, Andrea Caldera, Danyal Aytekin
+usemathjax: true
+categories:
+  - nx-plugin
+  - end-to-end-testing
+tags:
+  - open-source
+  - nx
+  - playwright
+date: 2022-08-05
+banner:
+  image: terminal.png
+---
+
+# Announcing our open-source project, Nx Plugins
+
+Today, we are excited to announce an open-source project called [Nx Plugins](https://github.com/marksandspencer/nx-plugins).
+
+## Introduction
+
+Nx Plugins is a [Nx](https://nx.dev) mono repo which consists of plugins which supports the Nx eco-system.
+
+Our plugins so far include:
+
+- [**nx-playwright**](https://www.npmjs.com/package/@mands/nx-playwright): an [Nx plugin](https://nx.dev/packages/nx-plugin) to add support to an Nx monorepo for Playwright testing using a native runner.
+
+## Motivation
+
+Our choice of framework for e2e testing is Playwright. There is already a plugin available for Playwright tests on NX but it does not support the native Playwright runner.
+
+The native Playwright runner gives you the following advantages:
+
+- Recommended by Playwright
+- Nicer syntax and built in assertions
+- Easier to find docs on playwright.dev
+
+## Why Open Source?
+
+Adoption for Nx as a mono repo build tool has significantly increased over time. However, there is often some extra configuration work required in order to use external libraries/tools.
+
+While working on an internal project where we were using Playwright for end-to-end testing, some manual configuration was required to integrate native Playwright with Nx.
+
+The team within M&S has come up with an internal project to create a plugin which simplifies integration of Nx with Playwright.
```

Comment:
just change it to "we" (as previous referred to as "we") , instead of "team within M&S"